### DISCUSSION TITLE
How should opemWEMI minimally be presented for approval?

### DISCUSSION THREAD

**Discussion by niklasl** [dcmi/usage#125] [Polls]
See https://github.com/dcmi/usage/issues/123

(We are assuming an RDF representation is mandatory.)

**Comment by annakasprzik** [comment]
I abstain from voting because it depends on the document: if it leaves out fine details of the specification that afterwards turn out to make all the difference then it is no good but otherwise one document is much easier to review, of course.

**Comment by aisaac** [comment]
I voted "Separate primer and formal specification" as a separate primer could be a good place for putting useful material (for instance https://github.com/dcmi/openwemi/issues/88) that would otherwise overload the 'real' specification.

**Comment by kcoyle** [comment]
Does "formal specification" mean a document? or is the RDF vocabulary a formal specification? If the former, then we would have three documents:
1. Formal specification
2. RDF vocabulary
3. Primer

**Comment by philbarker** [comment]
I did not vote. I don't have a strong preference for the number of documents, so long as it is clear what is normative and what is informative, and which document is authoritative should there be any differences (accidental or otherwise).

**Comment by kcoyle** [comment]
The group thinks that the [HTML file ](https://dcmi.github.io/openwemi/ns/openWEMI.html) is the formal specification; the primer is informative; the ttl file is the authoritative vocabulary. Is this what the UB thinks is needed?
